### ISSUE TITLE
Typhoeus-related error after killing server

### ISSUE THREAD

**Issue by wjdhamilton** [neo4jrb/neo4j#1377]
Background:

I have a model with the following properties:

```ruby
title: String
image: String
story: String
posted_on: String
```

`posted_on` represents a date. 

When trying to run the following erroneous cypher query (note the incorrect field name): 

```cypher
CYPHER
  MATCH (result_newsitem:`NewsItem`)
  WHERE (result_newsitem.started_on IN RANGE({result_newsitem_started_on_range_min}, {result_newsitem_started_on_range_max}))
  RETURN result_newsitem | {:result_newsitem_started_on_range_min=>-62167392000, :result_newsitem_started_on_range_max=>1493809765}
```

The neo4j server hung and could not be contacted.

I killed the neo4j process and restarted the server. Since then, I get the following error whenever I try to start my web app:

`The option: disable_ssl_peer_verification is invalid. Please try ssl_verifypeer instead of disable_ssl_peer_verification.`

Stack trace:

```ruby
        method = "#{key}="
        unless respond_to?(method)
          raise Errors::InvalidOption.new(key)
        end
        send(method, value)
      end
```

in:

```
ethon (0.10.1) lib/ethon/easy.rb:238:in `block in set_attributes'
ethon (0.10.1) lib/ethon/easy.rb:235:in `each_pair'
ethon (0.10.1) lib/ethon/easy.rb:235:in `set_attributes'
ethon (0.10.1) lib/ethon/easy/http/actionable.rb:99:in `setup'
ethon (0.10.1) lib/ethon/easy/http/get.rb:17:in `setup'
ethon (0.10.1) lib/ethon/easy/http.rb:39:in `http_request'
typhoeus (1.1.2) lib/typhoeus/easy_factory.rb:81:in `get'
typhoeus (1.1.2) lib/typhoeus/hydra/addable.rb:19:in `add'
typhoeus (1.1.2) lib/typhoeus/hydra/memoizable.rb:41:in `add'
typhoeus (1.1.2) lib/typhoeus/hydra/cacheable.rb:10:in `add'
typhoeus (1.1.2) lib/typhoeus/hydra/block_connection.rb:30:in `add'
typhoeus (1.1.2) lib/typhoeus/hydra/stubbable.rb:23:in `add'
typhoeus (1.1.2) lib/typhoeus/hydra/before.rb:27:in `add'
typhoeus (1.1.2) lib/typhoeus/hydra/queueable.rb:77:in `dequeue_many'
typhoeus (1.1.2) lib/typhoeus/hydra/runnable.rb:14:in `run'
typhoeus (1.1.2) lib/typhoeus/hydra/memoizable.rb:51:in `run'
faraday (0.12.1) lib/faraday/adapter/typhoeus.rb:23:in `perform_request'
faraday (0.12.1) lib/faraday/adapter/typhoeus.rb:14:in `call'
faraday_middleware (0.10.1) lib/faraday_middleware/response_middleware.rb:30:in `call'
faraday_middleware (0.10.1) lib/faraday_middleware/request/encode_json.rb:23:in `call'
faraday (0.12.1) lib/faraday/rack_builder.rb:139:in `build_response'
faraday (0.12.1) lib/faraday/connection.rb:386:in `run_request'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:129:in `block (2 levels) in request'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:187:in `wrap_connection_failed!'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:128:in `block in request'
neo4j-core (7.1.0) lib/neo4j/core/instrumentable.rb:29:in `block (3 levels) in instrument'
activesupport (5.0.2) lib/active_support/notifications.rb:164:in `block in instrument'
activesupport (5.0.2) lib/active_support/notifications/instrumenter.rb:21:in `instrument'
activesupport (5.0.2) lib/active_support/notifications.rb:164:in `instrument'
neo4j-core (7.1.0) lib/neo4j/core/instrumentable.rb:29:in `block (2 levels) in instrument'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:127:in `call'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:127:in `request'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:141:in `get'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session/adaptors/http.rb:62:in `constraints'
neo4j-core (7.1.0) lib/neo4j/core/cypher_session.rb:36:in `block (2 levels) in <class:CypherSession>'
neo4j-core (7.1.0) lib/neo4j/core/label.rb:100:in `constraints'
neo4j-core (7.1.0) lib/neo4j/core/label.rb:124:in `constraint?'
neo4j (8.0.16) lib/neo4j/migrations/runner.rb:17:in `initialize'
neo4j (8.0.16) lib/neo4j/migrations.rb:13:in `new'
neo4j (8.0.16) lib/neo4j/migrations.rb:13:in `check_for_pending_migrations!'
neo4j (8.0.16) lib/neo4j/migrations/check_pending.rb:13:in `call'
actionpack (5.0.2) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.2) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.2) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.2) lib/active_support/callbacks.rb:90:in `run_callbacks'
```
Searching for this specific error bring up an issue in Typhoeus, which is mentioned in the stack trace. 

Restarting the computer doesn't help.

I can still contact `localhost:7474/webbrowser`, but not using https.

I suspect an https issue but I don't know what to do next, other trying to delete the db and start again.

### Runtime information:

Neo4j database version: 3.1.1
`neo4j` gem version: 8.0.16
`neo4j-core` gem version: 7.1.0

**Comment by wjdhamilton** [comment]
Update: I tried reinstalling the db using community-latest and I still have the same problem. I have also updated Rails to 5.1.0 from 5.0.3 in case that had something to do with it. No luck with that either.

**Comment by wjdhamilton** [comment]
OK, I managed to fix it by following the usual advice on other repos for this type of error. I added an initialiser which with the following code:

```ruby
require "typhoeus"
require "typhoeus/adapters/faraday"
```

This sorted it.

**Comment by cheerfulstoic** [comment]
Ah, thanks for investigating this!  I'll add the `require`s in there and see if that fixes it

**Comment by cheerfulstoic** [comment]
Released version 7.1.1 of the `neo4j-core` gem which I think should allow you to not have to do the `require`s yourself.  Please let us know if that doesn't work and thanks again!
